cypress/e2e/drag-and-drop.feature

```diff
@@ -26,3 +26,22 @@ Feature: Test container drag and drop
     When I drag "#network2" onto "#network1 .component-container"
     Then I expect "#server2" to have "#network2 .component-container" as parent
     Then I expect "#network2" to have "#network1 .component-container" as parent
+
+  Scenario: Dragging a workflow steps should re-order them
```

Comment:
Add tests :
- add for vertical workflow
- move component from workflow 1 to workflow 2
- remove all component of workflow and add one